### PR TITLE
feat(package): rename packages

### PR DESCRIPTION
**Pull request by nichonien**
This PR updates the repo and package names to be more generic as mentioned below.

iam-roles               -> ew-credentials
onchain-role-enrolment  -> ew-onchain-claims
role-governance         -> ew-credential-governance
role-vc-verification    -> ew-vc-verification

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@energyweb/iam-roles",
+  "name": "ew-credentials",
```
Comment (jrhender) [inline package.json:2]: IMO, it still makes sense to have the packages namespaced under `@energyweb`. What do you think @nichonien ?
```suggestion
  "name": "@energyweb/ew-credentials",
```

**Review by jrhender — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nichonien) [reply package.json:2]: @jrhender I thought of it but then `@energyweb/ew-credential-governance` is a long package name. Also, `ew` does resemble `energyweb` in a shorter way/form.

Comment (jrhender) [reply package.json:2]: Hmm, IMO, all of energyweb's packages should be scoped under `@energyweb`. I can see how it seems a bit redundant but one could argue that `@energyweb` refers to the organization publishing the package and `ew` refers to the fact that we are doing credential governance in our more specific way.
If I had to chose between `@energyweb` and `ew`, I would chose `@energyweb`. So maybe the package name should be `@energyweb/credentials`. @mhrsntrk @jcmelati what are your thoughts here?

Comment (jrhender) [reply package.json:2]: Ok, after discussing with @mhrsntrk in standup, we will proceed with `@energyweb/credentials`

**Pull request merged**